### PR TITLE
bug 1616435 custom CA can contain only one cert

### PR DESCRIPTION
**Pull request by kalexand-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1616435

### CODE REVIEW

**Comment by kalexand-rh** [comment]
@gpei, will you PTAL?

@DanyC97, thanks for the suggestion.

@simo5, do you think that we need to document creating a custom cert, or do most admins usually know how to do this?

**Comment by simo5** [comment]
@kalexand-rh I do not think we should be in the business of trying to teach admins how to do certs.
There is plenty of documentation available on that for admins that want to learn.

**Comment by kalexand-rh** [comment]
Thanks @simo5! If you have a preferred method of generating certs that's doc'd in a different Red Hat collection, please let me know.

**Comment by kalexand-rh** [comment]
@gpei, does this change apply to only 3.10 and 3.11?

**Comment by kalexand-rh** [comment]
@openshift/team-documentation PTAL

**Comment by DanyC97** [comment]
@simo5 i would suggest try to look from a different angle which is not about _teaching admins how to create certs_.

If you look in the comment https://github.com/openshift/openshift-ansible/issues/9710#issuecomment-415520631 you can see exactly where the confusion is. 

And over there the key confusion was: what `certFile` should contain for the 2 use cases:
* just with CA root
* CA root and intermediate CA root

If you have a quick look around all the comments opened on openshift-ansible repo ( some are being referenced in the issue i've raised) you can see where my suggestion is coming from ...

**Comment by gpei** [comment]
@kalexand-rh seen from PR https://github.com/openshift/openshift-ansible/pull/8142/,  it should apply to 3.11, 3.10 and 3.9

**Comment by kalexand-rh** [comment]
@openshift/team-documentation PTAL

File: install_config/redeploying_certificates.adoc [hunk 1]
```diff
@@ -327,17 +332,27 @@ file. To use the current CA, skip this step.
 # playbook.
 openshift_master_ca_certificate={'certfile': '</path/to/ca.crt>', 'keyfile': '</path/to/ca.key>'}
 ----
-+
-If the CA certificate is issued by an intermediate CA, the bundled certificate must contain
-the full chain (the intermediate and root certificates) for the CA in order to validate child certificates.
-+
-For example:
+.. To specify a CA certificate that is issued by an intermediate CA:
+... Create bundled certificate that contains the full chain of intermediate and
```
Comment (ahardin-rh) [inline install_config/redeploying_certificates.adoc]: a bundled certificate

**Comment by ahardin-rh** [comment]
@kalexand-rh Just one comment from me. Otherwise, looks good!

**Comment by kalexand-rh** [comment]
Thanks @ahardin-rh!

**Pull request merged**